docs/general/reverse-proxy.md

```diff
@@ -0,0 +1,24 @@
+# Reverse Proxy
+
+Within TrueCharts our aim is to make it as easy as possible to secure your Apps. To support this we Supply a Traefik "Reverse Proxy", which has been preconfigured to provide secure and fast connections.
+
+To use Traefik as a Reverse Proxy, all you have to do is enable "Reverse Proxy" in the App of your choice and fill out a little form.
+
```

Comment:
```suggestion
### Setting up Traefik

The Traefik App takes over ports 80 and 443, which is likely to be where the TrueNAS SCALE UI is listening. To avoid losing access to the UI after Traefik starts up, go to System Settings -> General and edit the GUI settings to listen on a different port.

```

Reply:
Nope, this is traefik config.
It needs to go inside charts/traefik/2.0.0/docs

It has nothing to do with the general reverse-proxy configuration thats available on all Apps.

Reply:
Also: It takes those ports `by default` the new version in staging does offer the option to change the ports under advanced settings ;-)

Reply:
Yeah, fair point. With the changes to the questions, it's not so important, but I'll see what I can do instead.